modules/openfoam/src/OpenFOAM_Registry.txt

```diff
@@ -48,11 +44,11 @@ typedef	OpenFOAM/OpFM	ParameterType	IntKi	NnodesVel	-	-	-	"number of velocity no
 typedef	OpenFOAM/OpFM	ParameterType	IntKi	NnodesForce	-	-	-	"number of force nodes on FAST v8-OpenFOAM interface"	-
 typedef	OpenFOAM/OpFM	ParameterType	IntKi	NnodesForceBlade	-	-	-	"number of force nodes on FAST v8-OpenFOAM interface"	-
 typedef	OpenFOAM/OpFM	ParameterType	IntKi	NnodesForceTower	-	-	-	"number of force nodes on FAST v8-OpenFOAM interface"	-
-typedef ^       ^       ReKi            forceBldRnodes             {:}     "Radial location of force nodes"  -
-typedef ^       ^       ReKi            forceTwrHnodes             {:}     "Radial location of force nodes"  -
-typedef ^	^	ReKi	BladeLength	-	-	-	"Blade length (same for all blades)"	"m"
-typedef ^	^	ReKi	TowerHeight	-	-	-	"Tower height"	"m"
-typedef ^	^	ReKi	TowerBaseHeight	-	-	-	"Tower base height"	"m"
+typedef  OpenFOAM/OpFM	ParameterType	ReKi	forceBldRnodes	{:}	-	-	"Radial location of force nodes"  -
+typedef  OpenFOAM/OpFM	ParameterType	ReKi	forceTwrHnodes	{:}	-	-	"Radial location of force nodes"  -
```

Comment:
The description here is the same as `forceBldRnodes`. Maybe: 
"Vertical locations of tower force nodes (from bottom to top)"

Reply:
Good catch.